inspire_dojson/api.py

```diff
@@ -70,7 +71,9 @@ def marcxml2record(marcxml):
     marcjson = create_record(marcxml, keep_singletons=False)
     collections = _get_collections(marcjson)
 
-    if 'conferences' in collections:
+    if _is_from_cds(marcjson):
+        return hep.do(create_record_from_dict(cds2hep_marc.do(marcjson)))
```

Comment:
This bakes a concern about the origin of the MARCXML into a method that previously didn't care about it, stretching the abstraction a bit. It's not a blocker for merging, but I think we should consider moving this CDS MARCJSON -> INSPIRE MARCJSON logic to the only place in which we ingest CDS records, and keep this function agnostic.

(Note how this more limited abstraction doesn't allow us to harvest non-HEP records from CDS!)

Reply:
so you suggest not making `cds2hep_marc` part of `marcxml2record`, and let the caller do this additional step? We could still harvest non-hep collections by changing this line line to
`marcjson = create_record_from_dict(cds2hep_marc.do(marcjson)`, so that it then goes on with the rest of the logic.

Reply:
> We could still harvest non-hep collections by changing this line line to
> `marcjson = create_record_from_dict(cds2hep_marc.do(marcjson)`

Oh, I like this better than my proposal of having the caller handle it. But YAGNI, if we need this in the future we know how to do it.